src/nodes/inv_wishart.jl

```diff
@@ -0,0 +1,18 @@
+
+import StatsFuns: logπ
+import SpecialFunctions: loggamma
+import Distributions: InverseWishart
+
+@node InvWishart Stochastic [out, (ν, aliases = [df]), (S, aliases = [scale])]
+
+@average_energy InvWishart (q_out::Any, q_ν::PointMass, q_S::Any) = begin
+    d = dim(q_out)
+
+    m_q_ν = mean(q_ν)
+
+    return 0.5 * (
```

Comment:
Same comment here: [Avoid using floats for numeric literals in generic code when possible](https://docs.julialang.org/en/v1/manual/style-guide/#Avoid-using-floats-for-numeric-literals-in-generic-code-when-possible)